packages/mui-joy/src/Input/Input.tsx

```diff
@@ -167,6 +168,8 @@ const InputStartDecorator = styled('span', {
   '--Icon-margin': '0 0 0 calc(var(--Input-paddingInline) / -4)',
   display: 'inherit',
   alignItems: 'center',
+  paddingBlock: 'var(--unstable_Input-paddingBlock)', // for wrapping Autocomplete's tags
```

Comment:
Ok, we probably need to document css vars names :) `unstable` vs `_ComponentName` etc.

Reply:
We will likely need to document these as breaking changes going forward

Reply:
Yes, this is on my radar. I will create a PR that will refine the variable consistency across all components soon.